doc/api/api.raml

```diff
@@ -4,6 +4,12 @@ title: Metamaps
 version: v2.0
 baseUri: https://metamaps.cc/api/v2
 mediaType: application/json
+protocols: [ HTTPS ]
+documentation:
+  - title: Getting Started
+    content: !include pages/getting-started.md
+  - title: Endpoints
+    content: ""
```

Comment:
this makes an Endpoints link that quickly links past the long getting-started section to the real docs